pylint/utils/utils.py

```diff
@@ -57,6 +57,7 @@
     "suggestion-mode",
     "analyse-fallback-blocks",
     "allow-global-unused-variables",
+    "clear-cache-post-run",
```

Comment:
This shouldn't be used anymore as far as I know, so if it is necessary for tests to pass we have broken something 😅

Reply:
It wasn't, so I can take it out. I wondered if it was here for a public API of some sort. Will do.

Reply:
The `argparse` refactor should make all of this unnecessary!